### ISSUE TITLE
stopping minion kills only one process

### ISSUE THREAD

**Issue by idumbo** [saltstack/salt#40699]
### Description of Issue/Question
After starting appears two minion processes, but stopping kills only one
### Setup
Only master configured in /etc/salt/minion
### Steps to Reproduce Issue

> /etc/init.d/salt-minion start

```
root      2536  0.0  0.0 589764 32152 ?        Sl   15:04   0:02 /usr/bin/python2.6 /usr/bin/salt-minion -c /etc/salt -d
root      2537  0.0  0.0 493600 23320 ?        S    15:04   0:00  \_ /usr/bin/python2.6 /usr/bin/salt-minion -c /etc/salt -d
```

> /etc/init.d/salt-minion restart
> /etc/init.d/salt-minion restart
> /etc/init.d/salt-minion restart
> /etc/init.d/salt-minion restart
> /etc/init.d/salt-minion restart
```

~ # ps auxwf | grep salt
root      9547  0.0  0.0 107432   912 pts/1    S+   16:36   0:00  |       \_ grep salt
root      2537  0.0  0.0 493600 23320 ?        S    15:04   0:00 /usr/bin/python2.6 /usr/bin/salt-minion -c /etc/salt -d
root     17006  0.0  0.0 493616 23336 ?        S    16:34   0:00 /usr/bin/python2.6 /usr/bin/salt-minion -c /etc/salt -d
root     20853  0.0  0.0 493616 23336 ?        S    16:34   0:00 /usr/bin/python2.6 /usr/bin/salt-minion -c /etc/salt -d
root     21950  0.0  0.0 493608 23324 ?        S    16:35   0:00 /usr/bin/python2.6 /usr/bin/salt-minion -c /etc/salt -d
root     25129  0.0  0.0 493616 23336 ?        S    16:35   0:00 /usr/bin/python2.6 /usr/bin/salt-minion -c /etc/salt -d
root     32029  1.0  0.0 589772 31992 ?        Sl   16:35   0:00 /usr/bin/python2.6 /usr/bin/salt-minion -c /etc/salt -d
root     32044  0.0  0.0 493616 23332 ?        S    16:35   0:00  \_ /usr/bin/python2.6 /usr/bin/salt-minion -c /etc/salt -d
```
### Versions Report

salt-minion --versions
Salt Version:
           Salt: 2016.11.3
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.8.1
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.6.6 (r266:84292, Aug 18 2016, 15:13:37)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.5.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
 
System Versions:
           dist: centos 6.8 Final
        machine: x86_64
        release: 2.6.32-642.15.1.el6.x86_64
         system: Linux
        version: CentOS 6.8 Final

**Comment by idumbo** [comment]
Problem observed only when minion connected to master. With default minion config both processes was killed successfully.

**Comment by Ch3LL** [comment]
This is a duplicate of https://github.com/saltstack/salt/issues/40011 which has been fixed here: https://github.com/saltstack/salt/pull/40041 and will be included in 2016.11.4 which will be released this month.
